### ISSUE TITLE
proto/Google

### ISSUE THREAD

**Issue by MrMechis** [GoogleCloudPlatform/android-docs-samples#28]
Hi!,

Built the example no problem. 

Tried to remake it, in my own App, but needed "proto/Google" directory.   

Where did this come from?  

Thanks!

MrMechis

**Comment by cricket007** [comment]
This? 

https://github.com/GoogleCloudPlatform/android-docs-samples/tree/master/speech/Speech/app/src/main/proto/google

**Comment by MrMechis** [comment]
Yes, is that defined somewhere as package other than in the sample?

**Comment by cricket007** [comment]
You need to write your own `*.proto` files. Please notice this project is intended as a full sample, and it is using this plugin

https://github.com/google/protobuf-gradle-plugin

**Comment by MrMechis** [comment]
Yes, I understand this basic point, its not really my question.   I had no problem copying it and making it work.  But if I can't recreate or know where they come from its difficult if they upgrade for instance "speech/v1".  It would be helpful to know where this came from since the package is "package google.cloud.speech.v1;"    Does that make sense?  Is there a "proto" sample package?  Thanks!

**Comment by cricket007** [comment]
The speech api comes from here

https://godoc.org/google.golang.org/genproto/googleapis/cloud/speech

**Comment by MrMechis** [comment]
Excellent!  Thank you very much....
